install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -12,14 +12,17 @@ data:
   mesh: |-
     # Set the following variable to true to disable policy checks by the Mixer.
     # Note that metrics will still be reported to the Mixer.
-    disablePolicyChecks: false
+    disablePolicyChecks: {{ .Values.global.disablePolicyChecks }}
+
     # Set enableTracing to false to disable request tracing.
-    enableTracing: true
+    enableTracing: {{ .Values.global.enableTracing }}
+
     # Set accessLogFile to empty string to disable access log.
-    accessLogFile: "/dev/stdout"
+    accessLogFile: "{{ .Values.global.proxy.accessLogFile }}"
     #
     # To disable the mixer completely (including metrics), comment out
     # the following lines
+    # Deprecated: mixer is using EDS
     mixerCheckServer: istio-policy.{{ .Release.Namespace }}.svc.cluster.local:15004
```

Comment:
I think people wanted to configure this as well.. especially the svc.cluster.local as some ebay folks said that they have non default domains. so having this will break their system. In fact, Pilot also has an option to specify the cluster domain (I think). That needs to percolate

Reply:
You're right, still needed for that. Will remove the deprecated.

Maybe we should have an option to override the entire name ( so it can be pointed to DNS entry, for multicluster ) ?